pinot-spi/src/main/java/org/apache/pinot/spi/plugin/PluginManager.java

```diff
@@ -49,6 +89,7 @@ private PluginManager() {
    * @param directory
    */
   public void load(String pluginName, File directory) {
+    LOGGER.info("Trying to load plugin [{}] from location [{}]", pluginName, directory);
```

Comment:
Add another log to say that the plugin was loaded successfully. Maybe do this in the caller and catch the exception.